src/targets/services/categorize.js

```diff
@@ -8,7 +8,7 @@ import { BANK_DOCTYPE } from '../../doctypes'
 import { Model } from './helpers'
 
 export const categorize = async () => {
-  const { pretrained } = JSON.parse(process.env['COZY_PAYLOAD'] || [])
+  const { pretrained } = JSON.parse(process.env['COZY_PAYLOAD'] || "{ \"pretrained\": false }")
```

Comment:
I don't think `|| "{ \"pretrained\": false }"` is necessary: if it is not set in the `COZY_PAYLOAD`, `pretrained` will be undefined, and the `if (pretrained)` will evaluate to false either way

Reply:
If the payload is `unedfined`, `JSON.parse` fails. the argument should at least be `process.env['COZY_PAYLOAD'] || {}`

Reply:
Oh right, I didn't notice the parse included the default. Then `process.env['COZY_PAYLOAD'] || {}` looks good